trusted-committer/outline.md

```diff
@@ -0,0 +1,86 @@
+# Trusted Committer Learning Path Outline
+
+## Learning Goals
+
+- Definition of TC (why that name)
+- Understanding of the responsibilities of the Trusted Committer (TC)
+- Different ways to become TC
+- TC lifecycle (bringing on successors, sunsetting)
+- Useful skills/attitude/emotional maturity
+- TC antipatterns
+- Why is the TC role important for IS success
+
+## Target Audience
+
+- Current TCs
+- People who work/interface with TCs 
+    - Contributors
+    - Potential future TCs
+    - Managers of TCs
+
+## Outline
+
+### Definition
+
+- Glossary
+    - Contributor/Guest
+        - Responsible for "fitting in"/conforming to the host team and it's
+          expectations/processes.
+    - Product Owner (distinction btw. Guest and Host PO)
+    - TC
+        - TC has rights to push code closer to production than a contributor
+        - also known as _Host_
+        - has privileges that an admin or Apache committer would have but it is
+          a job, irrespective of the permissions. 
+        - Maintainer/Owner (conflict with _maintainer_ role in GitHub)
+        - Committer is not an IS term. Apache uses this term to denote a tech
+          leadership role (TC++)
+        - The TC role is defined by its responsibilities, rather than by its
+          privileges.
+
+- at PayPal, the discussion surrounding the name of the role was very
+  challenging.  i. e.: the name matters.
+
+### Responsibilities
+
+**Tech oriented**
```

Comment:
Maybe as a counterpoint state that TCs are _not_ responsible for writing guest contributions for them or rewriting contributions that come in not-to-standard.

Reply:
I agree that they are not responsible for that, indeed. But they should not be passive on that front either. I have always viewed it as part of my job as TC to assist people in making their contributions conform to the required standards. Usually by way of conversations in PRs, sometimes including pseudocode or sketches of algos. I generally avoided rewriting SW for contributors, but it did happen more than once.